input/otlp/traces_old.go

```diff
@@ -0,0 +1,665 @@
+package otlp
+
+import (
+	"fmt"
+	"net"
+	"net/netip"
+	"net/url"
+	"strconv"
+	"strings"
+
+	"github.com/elastic/apm-data/model"
+	"go.opentelemetry.io/collector/pdata/pcommon"
+	"go.opentelemetry.io/collector/pdata/ptrace"
+	semconv "go.opentelemetry.io/collector/semconv/v1.5.0"
+	"google.golang.org/grpc/codes"
+)
+
```

Comment:
Maybe add a warning comment here and in traces.go that any changes to these need to be kept in sync?